crates/rooch/src/commands/account/commands/balance.rs

```diff
@@ -66,30 +113,125 @@ impl CommandAction<Option<BalancesView>> for BalanceCommand {
+                    BalanceInfoViewUnion::Bitcoin(bitcoin_balance) => {
+                        balances_view.insert(
+                            bitcoin_balance.coin_info.name.to_string(),
+                            bitcoin_balance.into(),
+                        );
+                    }
+                    BalanceInfoViewUnion::Other(other_balance) => {
+                        balances_view.insert(
+                            other_balance.coin_info.coin_type.to_string(),
+                            other_balance.into(),
+                        );
+                    }
+                }
             }
             Ok(Some(balances_view))
         } else {
-            println!(
-                "{0: ^102} | {1: ^16} | {2: ^6} |  {3: ^32} ",
-                "Coin Type", "Symbol", "Decimals", "Balance"
-            );
-            println!("{}", ["-"; 68].join(""));
+            print_balance_table_header();
 
             for balance_info in balances {
-                println!(
-                    "{0: ^102} | {1: ^16} | {2: ^6} | {3: ^32} ",
-                    balance_info.coin_info.coin_type,
-                    balance_info.coin_info.symbol,
-                    balance_info.coin_info.decimals,
-                    balance_info.balance,
-                );
+                match balance_info {
+                    BalanceInfoViewUnion::Bitcoin(balance_info) => {
+                        print_balance_info(
+                            "Bitcoin".to_string(),
+                            balance_info.coin_info.symbol,
+                            balance_info.coin_info.decimals,
+                            balance_info.balance.to_string(),
+                        );
+                    }
+                    BalanceInfoViewUnion::Other(balance_info) => {
+                        print_balance_info(
+                            balance_info.coin_info.coin_type.to_string(),
+                            balance_info.coin_info.symbol,
+                            balance_info.coin_info.decimals,
+                            balance_info.balance.to_string(),
+                        );
+                    }
+                }
             }
 
             Ok(None)
         }
     }
 }
+
+async fn get_bitcoin_balance(
+    client: &Client,
+    address: RoochAddressView,
+) -> RoochResult<BitcoinBalanceInfoView> {
+    let total_balance: u64 = get_total_utxo_value(client, UnitedAddressView::from(address)).await?;
+    let bitcoin_info = BitcoinCoinInfoView {
+        name: "Bitcoin".to_string(),
+        symbol: "BTC".to_string(),
+        decimals: 8,
+    };
+    Ok(BitcoinBalanceInfoView {
+        coin_info: bitcoin_info,
+        balance: StrView(U256::from(total_balance)),
+    })
+}
+
+async fn get_total_utxo_value(
+    client: &Client,
+    address: UnitedAddressView,
+) -> Result<u64, anyhow::Error> {
+    let mut total_value: u64 = 0;
+    let mut cursor: Option<IndexerStateIDView> = None;
+
+    loop {
```

Comment:
If the user has a lot of UTXO records (for example, more than 10,000), the indexer query will be too frequent. We may consider maintaining a separate BTC Balance index.